ADDRESSING.md

```diff
@@ -20,18 +20,20 @@
 
 ## TL;DR
 
-If no native protocol handler is available, redirect to a gateway. The implementation should detect if a local IPFS node is available. In web browser contexts where a local IPFS node is present, use [subdomain gateway](https://docs.ipfs.io/how-to/address-ipfs-on-web/#subdomain-gateway) at `localhost`. If not, use public one such as `dweb.link`:
+If no native protocol handler is available, redirect to IPFS path at a gateway. The implementation should detect if a local IPFS node is available. In web browser contexts where a local IPFS node is present, use [subdomain gateway](https://docs.ipfs.io/how-to/address-ipfs-on-web/#subdomain-gateway) at `localhost`. If not, use public one such as `dweb.link`. 
+
+In both cases point at IPFS path first, to ensure gateway takes care of CID normalization into a DNS-safe form:
```

Comment:
> In both cases point at IPFS path first

Is this an instruction, or are we telling the reader that we are pointing to the IPFS path before we do anything else?

Reply:
It is instruction / "best practice".

Reply:
```suggestion
In either case, point at IPFS path first to ensure gateway takes care of CID normalization into a DNS-safe form:
```